SmartCharger.Data/Entities/Event.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SmartCharger.Data.Entities
+{
+    public class Event : BaseEntity
+    {
+        public DateTime StartTime { get; set; }
+        public DateTime? EndTime { get; set; }
+        public double? Volume { get; set; }
+
+        [ForeignKey(nameof(Charger))]
+        public int ChargerId { get; set; }
+        public virtual Charger Charger { get; set; }
+
+        [ForeignKey(nameof(Card))]
```

Comment:
Since Card has relation with User where it can only be linked to one user, why did you create a foreign key for both card and user?

Reply:
I gave this a lot of thought and tried to find edge cases. I've found a case where some user may delete his card and then give that same physical card to a different user. In that case, another user would be able to see his events (history of charging).

Reply:
That makes sense, thank you for the explanation. In that case there is no need for changes in that file.